t/t.go

```diff
@@ -626,6 +681,66 @@ func removeAllTestContainers() {
 	}
 }
 
+var loadPackages = []string{
+	"/systest/21million/bulk",
+	"/systest/21million/ludicrous",
+	"/systest/21million/live",
+	"/systest/1million",
+	"/systest/bulk_live/bulk",
+	"/systest/bulk_live/live",
+	"/systest/bgindex",
+	"/contrib/scripts",
+	"/dgraph/cmd/bulk/systest",
+}
+
+func isValidPackageForSuite(pkg string) bool {
+	if *suite == "load" {
+		return isLoadPackage(pkg)
+	}
+	return !isLoadPackage(pkg)
+}
+
+func isLoadPackage(pkg string) bool {
+	for _, p := range loadPackages {
+		if strings.HasSuffix(pkg, p) {
+			return true
+		}
+	}
+	return false
+}
+
+var datafiles = map[string]string{
+	"1million-noindex.schema": "https://github.com/dgraph-io/benchmarks/blob/master/data/1million-noindex.schema?raw=true",
+	"1million.schema": "https://github.com/dgraph-io/benchmarks/blob/master/data/1million.schema?raw=true",
+	"1million.rdf.gz": "https://github.com/dgraph-io/benchmarks/blob/master/data/1million.rdf.gz?raw=true",
+	"21million.schema": "https://github.com/dgraph-io/benchmarks/blob/master/data/21million.schema?raw=true",
+	"21million.rdf.gz": "https://github.com/dgraph-io/benchmarks/blob/master/data/21million.rdf.gz?raw=true",
+}
+
+func downloadDataFiles() {
+	if !*downloadResources {
+		fmt.Print("Skipping downloading of resources\n")
+		return
+	}
+	if *tmp == "" {
+		*tmp = os.TempDir()
+	}
+	x.Check(testutil.MakeDirEmpty([]string{*tmp}))
+	for fname, f := range datafiles {
```

Comment:
Can be parallelised

Reply:
Thats a great point. I will take this as separate task. I have to fix one more package containing bank test etc. I am hoping this might change a bit. I will fix in that PR.